app/muiTheme.tsx

```diff
@@ -0,0 +1,7 @@
+import getMuiTheme from "material-ui/styles/getMuiTheme";
+// colors should be same as in common/sass/variables.scss
+export default getMuiTheme({
+  palette: {
+    primary1Color: "#d5e20f",
```

Comment:
this is totally fine but I found webpack loader that can do load sass variables :D https://github.com/nordnet/sass-variable-loader but let's dont over compilcate things for now